gateway/cli/src/main.rs

```diff
@@ -99,7 +99,8 @@ async fn main() -> anyhow::Result<()> {
     TracingSetup::default().init()?;
 
     let cli = Cli::parse();
-    let client = || GatewayRpcClient::new(cli.address, cli.rpcpassword);
+    let versioned_api = cli.address.join("v1")?;
```

Comment:
can we move this "v1" into a const or config maybe? it's hardcoded in like 5 places